kube/services/revproxy/00nginx-config.yaml

```diff
@@ -80,6 +150,8 @@ data:
           #   not set headers ... http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_set_header
           #
           proxy_set_header   Authorization "$access_token";
+          proxy_set_header   X-Forwarded-For "$http_x_forwarded_for";
+          proxy_set_header   X-UserId "$userid";
           #add_header X-debug-auth "$access_token"; #- only do this for quick and dirty client-side debugging
```

Comment:
can remove this now?

Reply:
Need to pass the IP address data and user data on to the child services, so they can log it ...